app/component/navigation/toggle-map-tracking.cjsx

```diff
@@ -0,0 +1,25 @@
+React           = require 'react'
+Icon            = require '../icon/icon'
+cx              = require 'classnames'
+
+
+class ToggleMapTracking extends React.Component
+  @propTypes:
+    enableMapTracking: React.PropTypes.func.isRequired
+    disableMapTracking: React.PropTypes.func.isRequired
+    tracking: React.PropTypes.bool.isRequired
+
+  handleClick: =>
```

Comment:
Couldn't there just be one function passed from the parent that we could directly use as the click handler?